docs/src/demos/hooks/use-mouse/usage.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useMantineTheme, Text, Code } from '@mantine/core';
+import { useMouse } from '@mantine/hooks';
+
+const code = `
+import { useMantineTheme, Text, Code } from '@mantine/core';
+import { useMouse } from '@mantine/hooks';
+
+function Demo() {
+  const theme = useMantineTheme();
+  const { x, y } = useMouse();
+
+  return (
+    <Text align="center" style={{ marginTop: theme.spacing.sm }}>
+      Mouse coordinates <Code>{\`{ x: \${x}, y: \${y} }\`}</Code>
+    </Text>
+  );
+}`;
+
+function Demo() {
+  const theme = useMantineTheme();
+  const { x, y } = useMouse();
+
+  return (
+    <Text align="center" style={{ marginTop: theme.spacing.sm }}>
```

Comment:
You should leave margin here though for demo to look nice. General rule – remove all irrelevant presentational styles and properties from demo code but leave them in demo itself